ArduSub/fence.cpp

```diff
@@ -41,7 +41,7 @@ void Sub::fence_check()
             //            }
         }
 
-        // log an error in the dataflash
+        // log an error
```

Comment:
Seriously, just remove this sort of comment.  There are several like it and they add absolutely nothing.

Reply:
removed a bunch of them